### PR TITLE
Disable compilation of sundials and scikits.odes on mac

### PR DESCRIPTION
**Pull request by tlestang**
# Description
Since 03/02/2021 the scheduled "PyBaMM" workflow fails on mac for all python versions.
See https://github.com/pybamm-team/PyBaMM/actions/runs/532703527
The logs indicate that CMake cannot find the fortran compiler, and therefore Sundials isn't compiled... and the installation of scikits.odes fails.

Until a solution is found, this disables the compilation of sundials, as well as the installation of scikits.odes, on macos.

### CODE REVIEW

**Comment by tinosulzer** [comment]
Passed the mac tests by skipping every test 😂

**Comment by tlestang** [comment]
Thanks @tinosulzer for noticing that. I think it looks ok now

File: .github/workflows/benchmark_on_PR.yml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 name: Run benchmarks on PR
```
Comment (tinosulzer) [inline .github/workflows/benchmark_on_PR.yml:1]:
```suggestion
name: Run benchmarks on push
```

**Review by tinosulzer — APPROVED** [review]
awesome, thanks @tlestang

**Pull request merged**